doc/api/Renter.md

```diff
@@ -61,7 +61,17 @@ returns the current settings along with metrics on the renter's spending.
       // contract is scheduled to end, the contract is renewed automatically.
       // Is always nonzero.
       "renewwindow": 3024 // blocks
-    }
+    }, 
+    // MaxUploadSpeed by defaul is unlimited but can be set by the user to 
+    // manage bandwidth
+    "maxuploadspeed":     1234, // BPS
+
+    // MaxDownloadSpeed by defaul is unlimited but can be set by the user to 
+    // manage bandwidth
+    "maxdownloadspeed":   1234, // BPS
+
+    // The DownloadCacheSize is 
+    "downloadcachesize":  4   
```

Comment:
incomplete docstring

also I would use `bytes per second` instead of `BPS` for clarity